pkg/networks/usernet/client.go

```diff
@@ -29,7 +29,7 @@ func (c *Client) UnExposeSSH(sshPort int) error {
 }
 
 func (c *Client) ResolveAndForwardSSH(vmMacAddr string, sshPort int) error {
-	timeout := time.After(1 * time.Minute)
+	timeout := time.After(3 * 500 * time.Millisecond)
```

Comment:
This looks wrong. You have reduced the time to 1.5s.
Also there is clash with ticker.

Eg: ticker gets called for every 500ms. Timeout being 3*500ms. Only one of the channel will get called on. And other gets discarded. In your case it is working because timeout is ignore.

I would prefer to keep timeout maybe 2 * time.Minute.

Reply:
sorry, my mistake. 2 * time.Minute sounds good，thank you.